### PR TITLE
VACMS-4285: upgrade password policy modules

### PR DESCRIPTION
**Pull request by acrollet**
## Description

See #4285 

## Testing done

Manual testing, added e2e test for password strength.

## QA steps

As a user with the content admin role
1. Log in and attempt to change the user password
1. Then validate Acceptance Criteria from issue
- [ ] Weak passwords are not allowed
- [ ] A new strong password can be saved

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -83,7 +83,7 @@
         "drupal/menu_item_extras": "^2.7",
         "drupal/menu_link_attributes": "^1.1",
         "drupal/metatag": "^1.13",
-        "drupal/migrate_plus": "^4.0",
+        "drupal/migrate_plus": "5.1",
```
Comment (ndouglas) [inline composer.json:86]: Was this intended to be part of this PR?

Comment (acrollet) [reply composer.json:86]: whoops, no, thank you!

**Comment by acrollet** [comment]
@oksana-c this PR will have the effect of instituting strong password requirements when updating a password. ([slack thread for background](https://dsva.slack.com/archives/CT4GZBM8F/p1614705700105300)) Do you think we need a user-facing announcement?

**Review by ndouglas — APPROVED** [review]
Great test, confirmed it complains when I have 💩 passwords.

**Comment by acrollet** [comment]
> @oksana-c this PR will have the effect of instituting strong password requirements when updating a password. ([slack thread for background](https://dsva.slack.com/archives/CT4GZBM8F/p1614705700105300)) Do you think we need a user-facing announcement?

Documenting that we discussed on scrum and concluded that no announcement is necessary.

**Review by ndouglas — DISMISSED** [review]
Couldn't change password to a weak password, got appropriate errors when I tried, and was able to change to a strong password.

giphy

**Pull request merged**